src/edu/sjsu/fwjs/Expression.java

```diff
@@ -45,20 +43,6 @@ public Value evaluate(Environment env) {
 }
 
 /**
- * A print expression.
- */
-class PrintExpr implements Expression {
-    private Expression exp;
-    public PrintExpr(Expression exp) {
-        this.exp = exp;
-    }
-    public Value evaluate(Environment env) {
-        Value v = exp.evaluate(env);
```

Comment:
Hmm, it may be unavoidable to do this. Originally I was thinking it might be nice to make print be a function. This way it could be passed around as an argument to other higher-order functions. But since we don't have polymorphism we would need one print function for every type, which is untenable.

Therefore, let's add this back in.